src/grading-settings/assignment-section/index.jsx

```diff
@@ -76,8 +76,8 @@ const AssignmentSection = ({
             <ol className="course-grading-assignment-items p-0 mb-4">
               <AssignmentTypeName
                 value={gradeField.type}
-                errorEffort={errorList[`${type}-${idx}`]}
-                onChange={(e) => handleAssignmentChange(e, idx)}
+                errorEffort={errorList[`${type}-${gradeField.id}`]}
```

Comment:
optional: move these assignment and below out of return scope to separate variable